src/Infrastructure/IO/src/ESMCI_PIO_Handler.C

```diff
@@ -1559,7 +1559,7 @@ void PIO_Handler::attPackPut (
         return;
     }
     int size = (int)(jcurr.size());
-    ESMC_TypeKind_Flag att_type = ESMCI::json_type_to_esmf_typekind(jcurr, true);
+    ESMC_TypeKind_Flag att_type = ESMCI::json_type_to_esmf_typekind(jcurr, true, false); //tdk: figure out what to do here with the 32bit flag
```

Comment:
Does anything need to happen here before this goes in?

Reply:
Yes, I had not forgotten about it. Planning on addressing it today.